pkg/kubelet/cm/qos_container_manager_linux.go

```diff
@@ -209,12 +236,17 @@ func (m *qosContainerManagerImpl) UpdateReserve() error {
 		},
 	}
 
+	if err := m.setCPUReserve(qosConfigs); err != nil {
```

Comment:
The idea was that this call would be made from the switch statement below. I see that you are wanting to set `MinShares` on BE regardless of whether or not any cpu= option was provided to `--experimental-qos-reserve-limits`, which kinda works against the flow of the code here.

What would you think about having a function that set `MinShares` for BE from `Start()` since that won't ever need updating, and then have a separate function that updates the Burstable shares according to the reserve percentage the the users specifies, which currently isn't being used to calculate the Burstable share right now (why?).